types/smallwei__avue/element-plus.d.ts

```diff
@@ -0,0 +1,99 @@
+// import type { VNode } from 'vue';
+// import type { RuleItem } from 'async-validator';
```

Comment:
ditto about dead code imports
you can write proper comments though and keep them inline in the code